.github/workflows/check.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         run: |
           # https://github.com/opensearch-project/opensearch-build/issues/4191
           chown -R ci-runner:ci-runner `pwd`
-          su ci-runner -c "source /etc/profile.d/java_home.sh && ./gradlew check -Dorg.gradle.java.home=/opt/java/openjdk-${{ matrix.java }}"
+          su ci-runner -c "source /etc/profile.d/java_home.sh && ./gradlew test check -Dorg.gradle.java.home=/opt/java/openjdk-${{ matrix.java }}"
```

Comment:
it's a simple refactor right?

Reply:
yes, for some reasons the `test` task is not executed as part of the `./gradlew check`, only `integTest` (for now adding it explicitly)